kubernetes/helm-charts/buildfarm/templates/configmap.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "buildfarm.fullname" . }}-config
+data:
+  config.yml: |-
+    {{- range $key, $value := .Values.config }}
+    {{- if kindIs "map" $value }}
+    {{- else }}
+    {{ $key }}: {{ $value }}{{- end }}
+    {{- end }}
+    backplane:
+      {{- if .Values.redis.enabled }}
+      redisUri: "redis://{{ printf "%s-redis-master.%s" (include "redis.fullname" .) (.Release.Namespace) }}:{{ "6379" }}"
```

Comment:
Others have indicated that there may need to be a scheme control here (for rediss://), and the port and presumably operator convention are hard coded.

Reply:
ok. I will update the PR later

Reply:
@werkt The PR has been updated. Please help to review it, thanks~